foundry/foundry.py

```diff
@@ -857,6 +857,53 @@ def toTorch(self, raw=None, split=None):
             return FoundryDataset_Torch(inputs, targets)
         else:
             raise NotImplementedError
+
+    def toTensorflow(self, raw=None, split=None):
```

Comment:
Check out the Google Python Style guide: they discourage camelCase and recommend underscore_style. I'd recommend keeping to that here.